### PR TITLE
Add alpha-beta-v1 search strat

### PR DESCRIPTION
**Pull request by Samuel-Lewis**
Tokens: 590

Moves gen1 to EvilBot. Beats gen1 90% of the time.

### CODE REVIEW

File: Chess-Challenge/src/API/Board.cs
Comment (Samuel-Lewis) [inline Chess-Challenge/src/API/Board.cs:1]: Can ignore. This is just linting.

File: Chess-Challenge/src/Evil Bot/EvilBot.cs
Comment (Samuel-Lewis) [inline Chess-Challenge/src/Evil Bot/EvilBot.cs:1]: Moves gen1 into evilbot

File: Chess-Challenge/src/My Bot 1_0_0/MyBot.cs
Comment (Samuel-Lewis) [inline Chess-Challenge/src/My Bot 1_0_0/MyBot.cs:1]: Copy of gen1

**Review by Jacobious52 — APPROVED** [review]
lgtm. we can probably look at using [NegaMax](https://www.chessprogramming.org/Negamax#:~:text=a%20common%20way%20of%20implementing,min(%2Da%2C%20%2Db)) later for score so we can trim down the if / else needed in the search.

**Pull request merged**